### ISSUE TITLE
How can i get git revision number?

### ISSUE THREAD

**Issue by netpyoung** [vaab/gitchangelog#49]
when i try to get short sha to use CHANGELOG.md, but i can't get that info.

https://github.com/vaab/gitchangelog/blob/master/gitchangelog.py#L891

``` python
            sections[matched_section].append({
                "author": commit.author_name,
                "subject": subject_process(commit.subject),
                "body": body_process(commit.body),
            })
```

how about add two version of sha info?

```
"sha1": commit.sha1,
"sha1_short": commit.sha1[:6],
```

**Comment by vaab** [comment]
Sorry for the very late answer, but somehow your issue went unnoticed. Wouldn't #18 solve your use case ? By providing directly the Commit, it would then provide you the ability to query ``sha1`` for instance.

May I add that you should be carefull about sha1_short which might not be simply sha1 truncated to 6 char as it happens that doubles occurs. This is probably for you here a side concern, and is indeed a separate problem.

**Comment by vaab** [comment]
Thank for your report. I have closed your issue because #18 was fixed and provides you with the raw commit info. And the last push adds also a solid ``sha1_short`` implementation to the commit object.
Any comment are welcome. Sorry for the delay, and thanks for the suggestion.
